tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py35,py36,py37,pypy2
+envlist = py27,py36,py37,py38,py39,pypy2
```

Comment:
This seems to also be adding support for newer versions. Can you create a separate PR for that? It should not be controversial (compared to dropping an old version) and we can merge it right away.

Reply:
Created
https://github.com/trinodb/trino-python-client/pull/72